libmamba/src/core/validate.cpp

```diff
@@ -194,6 +194,33 @@ namespace validate
         return ::mamba::hex_string(hash, MAMBA_MD5_SIZE_BYTES);
     }
 
+    std::string blake2sum(const fs::u8path& path)
+    {
+        size_t sum, n;
+        // blake2b_state S;
+        constexpr std::size_t BUFSIZE = 32768;
```

Comment:
Prefer `static constexpr` for these, see [there](https://stackoverflow.com/a/13867690) for details